libs/chlorophyll/scss/components/link/_index.scss

```diff
@@ -1,19 +1,17 @@
 @use 'mixins';
 @use 'sass:map';
 
-
```

Comment:
i think it would be nice if we where to use scoped classes eg. "seb-green-link", that whay it's less prone to leak styling, also it is easier to understad where the styling is coming from when inspecting elements in the browser

Reply:
Green should always be scoped to .use-green to prevent styling, that's the idea right now.

Having tailwind style classes might cause developers to "implemented their own component" with utility classes and future changes to design system could cause breaks. Having those cases as specific components is better to get the feedback loop in place.

Reply:
Just to add a bit of background here, it was decided in discussions early on that one of the goals of Chlorophyll should be to make markup look correct "by default".

I.E. the following snippet of markup should render a correct SEB-styled document, provided you load the Chlorophyll CSS:
```html
<h1>Hello world</h1>
<p>Lorem ipsum <a href="/url">dolor</a> sit amet</p>
<button>Do something!</button>
```

In other words, the lack of scoped classes is "by design" :)

But I think this is a discussion we should keep having. It seems likely that this approach will keep causing problems in all sorts of scenarios in the future. Perhaps it would be good enough that the React and Angular components looks correct by default, and that Chlorophyll requires prefixed classes for everything.

Reply:
I agree, we should keep having this discussion! 

It will cause problems in all sorts of scenarios but I think that could be a good thing. 

If a such a layout/styling problems occurs: 
1. Either the markup is faulty and needs to be corrected or 
3. Green needs to get that contribution/feedback in order to update styles. And nothing gets fixed faster than things that doesn't **look** good 😄